src/helpers/photoUpload.js

```diff
@@ -0,0 +1,19 @@
+import uploader from '../database/config/cloudinary';
+
+
+const profileImage = async (req) => {
+    try {
+        const tmp = req.files.image.tempFilePath;
+        const Result = await uploader.upload(
+            tmp,
+            { folder: 'winners-c8-bn-be' },
```

Comment:
I would also propose that the cloudinary folder name would be like  '`winners-c8-bn-be/profile-pictures`' or something which is not just '`winners-c8-bn-be`', so that the file paths of this feature's images will be more clear & distinguishable from other features that will use cloudinary.

Reply:
he must flow the naming conventions

Reply:
yeah i did